### PR TITLE
Added my blog with an image filename.

### PR DESCRIPTION
**Pull request by pmsbir**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — COMMENTED** [review]
Just upload this image in the img/blogs folder and you are done !

File: _data/blogs.yml [hunk 1]
```diff
@@ -197,3 +197,8 @@
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
   author_name: Saarthak Chaturvedi
+
+- url: https://limitless497584285.wordpress.com/2017/12/27/coding-my-joy/
+  title: CODING- MY JOY
+  img: pmslife.jpg
```
Comment (anshumanv) [inline _data/blogs.yml:203]: Please upload the image with this file name of 240x240.

**Comment by pmsbir** [comment]
I have uploaded it

File: _data/blogs.yml (hunk 1 above)
Comment (pmsbir) [reply _data/blogs.yml:203]: Here's the link
https://github.com/fossasia/gci17.fossasia.org/pull/573/files/63dddbfb9662c790e2cb0b6c7e1daa0f74c5a80b

**Comment by pmsbir** [comment]
This link defines it all:
https://github.com/fossasia/gci17.fossasia.org/commit/6c89faf41a328f5b0db1d2674cc0b483de1b6292

**Comment by anshumanv** [comment]
Please follow up in #573, closing this.

**Comment by pmsbir** [comment]
Can you please merge my pr quickly🙏🙏

**Comment by anshumanv** [comment]
Please make requested changes in the other PR.

**Pull request closed without merging**